linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -415,20 +415,20 @@ MOCKABLE_FUNCTION(, int, threadpool_schedule_work_item, THANDLE(THREADPOOL), thr
 
 - **SRS_THREADPOOL_LINUX_05_026: [** `threadpool_schedule_work_item` shall succeed and return 0. **]**
 
-### threadpool_destroy_work_item
+### threadpool_dispose_work_item
 
 ```c
-MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), threadpool, THREADPOOL_WORK_ITEM_HANDLE, threadpool_work_item);
+static void threadpool_dispose_work_item(THANDLE(THREADPOOL_WORK_ITEM), threadpool_work_item);
 ```
 
-`threadpool_destroy_work_item` Waits for completion of scheduled work items before freeing the context.
+`threadpool_dispose_work_item` Waits for completion of scheduled work items before freeing the context.
 
-**SRS_THREADPOOL_LINUX_05_029: [** If `threadpool` is `NULL`, `threadpool_destroy_work_item` shall fail. **]**
+**SRS_THREADPOOL_LINUX_05_029: [** [Deprecated] If `threadpool` is `NULL`, `threadpool_dispose_work_item` shall fail. **]**
 
-**SRS_THREADPOOL_LINUX_05_030: [** If `threadpool_work_item` is `NULL`, `threadpool_destroy_work_item` shall fail. **]**
+**SRS_THREADPOOL_LINUX_05_030: [** [Deprecated] If `threadpool_work_item` is `NULL`, `threadpool_dispose_work_item` shall fail. **]**
 
-**SRS_THREADPOOL_LINUX_05_033: [** `threadpool_destroy_work_item` shall wait for all pending work items to finish execution. **]**
+**SRS_THREADPOOL_LINUX_05_033: [** [Deprecated] `threadpool_dispose_work_item` shall wait for all pending work items to finish execution. **]**
 
-  - **SRS_THREADPOOL_LINUX_05_035: [** `threadpool_destroy_work_item` shall free the memory allocated to the work item of type `THREADPOOL_WORK_ITEM_HANDLE` created in `threadpool_create_work_item`. **]**
+  - **SRS_THREADPOOL_LINUX_05_035: [** `threadpool_dispose_work_item` shall free the memory allocated to the work item of type `THANDLE(THREADPOOL_WORK_ITEM)` created in `threadpool_create_work_item`. **]**
```

Comment:
This is not really done by the threadpool_dispose_work_item.
As a matter of fact, for Linux I think threadpool_dispose_work_item is probably not needed at all. A dispose function is needed if any actions need to be taken to free/dispose of something when the reference count reaches 0.

Reply:
Fixed.